pkg/cluster/sync.go

```diff
@@ -57,8 +57,8 @@ func (c *Cluster) Sync(newSpec *acidv1.Postgresql) error {
 		return err
 	}
 
+	c.logger.Debugf("syncing volumes using %q resize mode", c.OpConfig.StorageResizeMode)
```

Comment:
I believe that does not bring much to people unfamiliar with operator internals. "pvc / ebs resize" is also too generic to google. It would be better to log
```
syncing volumes using kubernetes-native PVC resize (%q mode)
```
or
```
syncing volumes by exec'ing into a pod and calling AWS API on its EBS volumes (%q mode)
```

Reply:
I've now added the word "storage" to make the link to the config option. This should be fine.